### DISCUSSION TITLE
Test des devs en cours de déploiement Docker

### DISCUSSION THREAD

**Discussion by hypsug0** [PnX-SI/GeoNature#2334] [General]
Salut,

Je viens aux nouvelles sur la partie Docker, @bouttier, où en est-tu? Je veux bien faire beta-testeur. J'ai cherché un docker-compose dans la branche feat/docker mais n'ai rien trouvé.

Merci!

**Comment by camillemonchicourt** [comment]
Les Dockerfiles ont été mergé et intégrés dans la version 2.11.
Le travail le plus à jour est ce qu'il y a dans la branche DEVELOP.
Voir https://github.com/PnX-SI/GeoNature/issues/2206

Il y a aussi les images publiées automatiquement : https://github.com/orgs/PnX-SI/packages?repo_name=GeoNature

Il y a un travail sur les Docker-compose ici, mais je ne sais pas bien où cela en est - https://github.com/PnX-SI/GeoNature-Docker-services/tree/docker-compose

**Reply by camillemonchicourt** [reply]
Le docker-compose complet et fonctionnel est disponible depuis la version 2.13 et disponible dans le dépôt dédié : https://github.com/PnX-SI/GeoNature-Docker-services/tree/docker-compose
